common_core.c

```diff
@@ -7,7 +7,7 @@ int cclient_get_txn_to_approve(const iota_client_service_t* const service,
       get_transactions_to_approve_req_new();
   get_transactions_to_approve_res_t* get_txn_res =
       get_transactions_to_approve_res_new();
-  get_txn_req->depth = 15;
+  get_transactions_to_approve_req_set_depth(get_txn_req, 3);
```

Comment:
How the depth is determined?

Reply:
Max is 15, min is 3. Usually it's fine to set depth to minimum, but default docs would show 15 intentionally.

Reply:
Insert the relevant code comment(s) about `depth`.